src/node.js

```diff
@@ -388,17 +389,17 @@
       tickDone();
     }
 
+    function PObj(c) {
+      this.callback = c;
+      this.domain = process.domain || null;
+    }
+
     function nextTick(callback) {
       // on the way out, don't bother. it won't get fired anyway.
       if (process._exiting)
         return;
 
-      var obj = {
-        callback: callback,
-        domain: process.domain || null
-      };
```

Comment:
[This](https://github.com/v8/v8-git-mirror/blob/8b33fa60691fe3d0d0e2ee6b751b89de6abec626/src/preparser.h#L2356-L2363) is the code that estimates the number of properties a function constructor will introduce.